<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="95" started-at="2016-05-11T03:16:02Z" finished-at="2016-05-11T03:16:02Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="95" started-at="2016-05-11T03:16:02Z" finished-at="2016-05-11T03:16:02Z">
      <class name="com.metacube.ipathshala.suite.Login.LoginByAdmin">
        <test-method status="FAIL" signature="createDriver()[pri:0, instance:com.metacube.ipathshala.suite.Login.LoginByAdmin@2f4a2e75]" name="createDriver" is-config="true" duration-ms="0" started-at="2016-05-11T08:46:02Z" finished-at="2016-05-11T08:46:02Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.metacube.ipathshala.suite.Login.LoginByAdmin.createDriver(LoginByAdmin.java:33)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:175)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:107)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createDriver -->
        <test-method status="SKIP" signature="passUrl()[pri:0, instance:com.metacube.ipathshala.suite.Login.LoginByAdmin@2f4a2e75]" name="passUrl" is-config="true" duration-ms="0" started-at="2016-05-11T08:46:02Z" finished-at="2016-05-11T08:46:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- passUrl -->
        <test-method status="SKIP" signature="loginByAdmin()[pri:0, instance:com.metacube.ipathshala.suite.Login.LoginByAdmin@2f4a2e75]" name="loginByAdmin" duration-ms="11" started-at="2016-05-11T08:46:02Z" finished-at="2016-05-11T08:46:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginByAdmin -->
      </class> <!-- com.metacube.ipathshala.suite.Login.LoginByAdmin -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
